Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11-slim@sha256:39cecc9cde774f9209ad26a0ab2dc4f5d10ba92d2913a835cebea3b402af8e8b
+
+# Configuración Poetry (https://python-poetry.org/docs#ci-recommendations)
+ENV POETRY_VERSION=1.3.2
+
+ENV VIRTUAL_ENV=/opt/poetry-venv
+RUN python3 -m venv $VIRTUAL_ENV
```

Comment:
¿Por qué haces esto y con qué usuario lo estás haciendo?

Reply:
:arrow_up: Si no habéis hecho ningún cambio respondiendo a las preguntas que os he hecho, para optimizar nuestro tiempo, os rogaría que no solicitárais revisión.

Reply:
Y sigues sin responder esto, ni el resto. Cuando lo hagas, por favor, pide revisión.